apps/website/.vuepress/theme/components/Home.vue

```diff
@@ -221,15 +221,15 @@
         <div cds-text="center" cds-layout="horizontal gap:lg align:fill col:start-2 col:10 m-t:xl">
           <div>
             <h3 cds-text="display" class="docs-red">2M+</h3>
-            <p>downloads since the beginning of Clarity</p>
+            <p cds-text="body" cds-layout="m-t:lg">downloads since the beginning of Clarity</p>
```

Comment:
For future reference, you can use vertical layouts to achieve this with `gap:lg`. That may be preferable in some cases. Here it's just a stylistic choice.